repository/main/settings.py

```diff
@@ -27,6 +27,8 @@
 
 ALLOWED_HOSTS = []
 
+# allow connections from any internal host
+ALLOWED_CIDR_NETS = ["10.0.0.0/8"]
```

Comment:
One doubt here @psschwei , must be this `net` configurable? Are we expecting that can be changed maybe?

Reply:
Good question. I'm not sure if it needs to be configurable... we'd want to allow access from within the cluster (`10.` addresses) and from users loading things into it (I'm assuming we'll have an API that makes an HTTP request). 

By configurable, I'm assuming you're meaning from an end user (and not someone building their own image from source).

(Lots of assumptions here... )

Reply:
> got it ... yeah, we'll probably need to tweak that for different environments going forward... for now, the 10. will cover the private IPs of a k8s cluster.

Understood!

> I'd recommend phasing out docker compose for local dev in favor of kind / rancher

I was mostly using `k8s` until now too but for convenience we were maintaining `docker-compose`. Probably is a discussion that we can have now that #306 is getting more attention, I totally agree 👍

Reply:
Seems like the preferred Django solution for production is to use a domain name to the allowed hosts, though not sure exactly how that would work on a HPC...